src/components/robot/pages/RobotDuplicatePage.tsx

```
@@ -0,0 +1,115 @@
+      saveButtonText={t("robot_duplication.buttons.duplicate")}
+      isLoading={isLoading}
+      showCancelButton={false}
+    >
+      <>
+        <Box style={{ display: "flex", flexDirection: "column" }}>
+          {robot && (
+            <>
+              <span>{t("robot_duplication.descriptions.purpose")}</span>
+              <br />
+              <span
+                dangerouslySetInnerHTML={{
```

Biome (2.4.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

(lint/security/noDangerouslySetInnerHtml)